prow/config/config.go

```diff
@@ -53,6 +54,10 @@ type Config struct {
 	// Defaults to "default".
 	PodNamespace string `json:"pod_namespace,omitempty"`
 
+	// LogLevel enables dynamically updating the log level of the
+	// standard logger that is used by all prow components.
+	LogLevel string `json:"log_level,omitempty"`
```

Comment:
Somehow indicate that it must be one of [these](https://github.com/sirupsen/logrus/blob/89742aefa4b206dcf400792f3bd35b542998eb3b/logrus.go#L36-L54). Maybe just explicitly list them here?

Reply:
done